store/src/java/com/zimbra/cs/filter/JsieveConfigMapHandler.java

```diff
@@ -39,6 +39,7 @@
     public static final String CAPABILITY_EREJECT    = "ereject";
     public static final String CAPABILITY_REJECT     = "reject";
     public static final String CAPABILITY_ENOTIFY    = "enotify";
+    public static final String CAPABILITY_EDITHEADER = "editheader";
```

Comment:
Please use CAPABILITY_EDITHEADER in the declaration of the "mCommandMap" parameter (line 65-66)